aten/src/ATen/code_template.h

```diff
@@ -78,10 +79,8 @@ struct TemplateEnv {
   }
 
  private:
-  [[noreturn]] void notFound(const std::string& k) const {
-    std::stringstream ss;
-    ss << "key not found: " << k;
-    throw std::logic_error(ss.str());
+  [[noreturn]] void notFound(std::string k) const {
+    throw std::logic_error("key not found: " + std::move(k));
```

Comment:
I don't think the move ctor actually does anything when appending so a ref /stringview is better

Reply:
const ref is better. string_view can't be appended to std::string.